src/managed/dir.proj

```diff
@@ -24,7 +24,7 @@
       <Output ItemName="buildCmdArgs" TaskParameter="BuildArgs" />
     </GetBuildArgsByFrameworks>
 
-    <Exec Command="$(DotnetToolCommand) build $(Args) %(buildCmdArgs.Identity)" 
+    <Exec Command="$(DotnetSdkToolCommand) build $(Args) %(buildCmdArgs.Identity)" 
```

Comment:
This doesn't work because of the way we are running msbuild.  On Windows, we are using the desktop msbuild in VS 2015, which doesn't support the new "SDK" projects.  On non-Windows, we are using the msbuild that comes with build-tools, which also doesn't support the new "SDK' projects.